addons/main/weapons/CfgWeapFAL.hpp

```diff
@@ -31,6 +31,8 @@ class hlc_rifle_C2A1: hlc_fal_base {
 };
 
 class hlc_rifle_FAL5061: hlc_fal_base {
+    ACE_barrelTwist = 304.8;
+    ACE_barrelLength = 458;
```

Comment:
Barrel twist is already defined in-game for all FALs - can you please remove them?

As far as I can tell, the 50.61 has a full length barrel (21 in or 533mm). Can you provide sources if it isn't the case?
```suggestion
    ACE_barrelLength = 533;
```